Sources/WriteFreely/WFClient.swift

```diff
@@ -57,10 +57,16 @@ public class WFClient {
         alias: String? = nil,
         completion: @escaping (Result<WFCollection, Error>) -> Void
     ) {
-        if token == nil && user == nil { return }
-        guard let tokenToVerify = token ?? user?.token else { return }
+        if token == nil && user == nil { completion(.failure(WFError.couldNotComplete)) }
```

Comment:
Do we still need to `return` here, or is that handled further down in the logic?

Reply:
Whoops! Good catch — fixed in 1213eef.

Reply:
:+1: There are also a few other spots where the `return` was dropped. I'll mark those, in case they still need to be there.